.github/ISSUE_TEMPLATE/issue-report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Issue report
+about: Create an issue to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the issue**
+A clear and concise description of what the issue is.
+
+**Environment (please complete the following information):**
+ - Host OS and version: [e.g. Oracle Linux 8.2 UEK6, macOS 10.14.6, Windows 10 Pro, ...]
+ - Vagrant version:  [e.g. 2.2.9]
+ - Vagrant provider:
+   - For VirtualBox:
+      - VirtualBox version: [e.g. 6.1.8r137981 -- run `vboxmanage -v`]
+   - For libvirt:
+     - QEMU version: [e.g. 3.1.0 -- run `rpm -q qemu-kvm`]
```

Comment:
Some `libvirt` hosts may not have `rpm`. Would `qemu -v` work?

Reply:
Yes I am aware of that -- I'll see if we can have something more generic for these two

Reply:
There isn't anything we can rely on cross-distributions.
I have rephrased...